scripts/make_lm_dir.py

```diff
@@ -123,20 +126,22 @@
 f = open(args.lm_dir + "/was_pruned", "w")
 print("false", file=f)
 f.close()
-
+clean_up_opt = "--clean-up false" if args.clean_up == "false" else ""
```

Comment:
Maybe the clean_up_opt should be set also in the else part. In case that we may decide to change the default value of --clean-up in get_objf_and_derivs.py. I think it is better to remove the dependence on other scirpts